downstream/modules/platform/con-aap-upgrade-prereq.adoc

```diff
@@ -0,0 +1,151 @@
+[id="con-aap-upgrade-prereq_{context}"]
+
+= Prerequisites for upgrading {PlatformNameShort}
+
+Before you begin to upgrade {PlatformNameShort}, ensure your environment meets the following node and configuration requirements.
+
+== Node requirements
+
+The following specifications are required for the nodes involved in the {PlatformNameShort} upgrade process:
+
+* 16 GB of RAM for controller nodes, database node, execution nodes and hop nodes.
+* 4 CPUs for controller nodes, database nodes, execution nodes, and hop nodes.
+* 150 GB+ disk space for database node.
+* 40 GB+ disk space for non-database nodes.
+* DHCP reservations use infinite leases to deploy the cluster with static IP addresses.
+* DNS records for all nodes.
+* Red Hat Enterprise Linux 8 or later 64-bit (x86) installed for all nodes.
+* Chrony configured for all nodes.
+
+== {ControllerNameStart} configuration requirements
+The following {ControllerName} configurations are required before you proceed with the {PlatformNameShort} upgrade process:
+
+.Configuring NTP server using Chrony
+
+Each {PlatformNameShort} node in the cluster must have access to an NTP server. Use the `chronyd` to synchronize the system clock with NTP servers. This ensures that cluster nodes using SSL certificates that require validation do not fail if the date and time between nodes are not in sync.
+
+This is required for all nodes used in the upgraded {PlatformNameShort} cluster:
+
+. Install `chrony`:
++
+----
+# dnf install chrony --assumeyes
+----
+. Open `/etc/chrony.conf` using a text editor.
+. Locate the public server pool section and modify it to include the appropriate NTP server addresses. Only one server is required, but three are recommended. Add the 'iburst' option to speed up the time it takes to properly sync with the servers:
```

Comment:
Should iburst be enclosed in backticks? That is, do you want the quotes to appear in the doc ?